content/post/sp-reflection-002.md

```diff
@@ -0,0 +1,41 @@
+
+A strong motivation for my research is to be able to better articulate _what_ and _why_ is 1RG Space so the notes make sense, but they also feel distant - this is maybe an obvious result of the time that has passed between writing those notes and sitting down to reflect upon them, but there's more. They excercise no traction on my curiosity.
+
+My mind wonders, uneasy, and unsure what direction to take. Something is missing.
+
+My natural inclination, when I start learning something new, is to to start from first principles. Then what's missing is a definition of _community_, one that is _personal_ as opposed to the wikipedia impersonal[^wikipedia]. I am convinced that definition holds the keys to the direction of this research.
+[^wikipedia]: _a social unit (a group of living things) with a shared socially significant characteristic, such as place, set of norms, culture, religion, values, customs, or identity._
+
+Much of my time in Italy was spent lounging in the sun and reading - a historical fantasy epic about student revolutions, colonial resistance, and the use of language and translation as the dominating tool of the British Empire[^babel], a journalistic account of the Sidewalk Labs debacle in Toronto[^sideways], and the philosophical manifesto of resistance to the attention economy[^nothing]
+[^babel]: [Babel, or the Necessity of Violence: an Arcane History of Oxford Translators](https://www.goodreads.com/book/show/57945316-babel) by R.F. Kuang
+[^sideways]: [Sideways: the city Google couldn't buy](https://www.goodreads.com/book/show/61167317-sideways) by Josh O'Kane
+[^nothing]: [How to do nothing: resisting the attention economy](https://www.goodreads.com/book/show/42771901-how-to-do-nothing) by Jenny Odell
+
+While none of the reads is explicitly related to this research, I can see a common thread that connects them to it. 
+
+Humor me[^spoilers].
+[^spoilers]: Note: what follows contains spoilers.
+
+I started reading _How to do Nothing_ before my vacation and I was captivated by the concept of [_resistance-in-place_](https://www.resistancein.place/) [^joy] as described by the author:
+[^joy]: I hope the reader finds as much joy I did in finding out that this website has no other purpose than being a quote from the book.
+
+> To resist in place is to make oneself into a shape that cannot so easily be appropriated by a capitalist value system [ ... ] embracing and trying to inhabit somewhat fuzzier or blobbier ideas: of maintenance as productivity [... ] recognizing and celebrating a form of the self [ ... ] whose identity doesn’t always stop at the boundary of the individual.
```

Comment:
I forgot this but I like it. And it can be framed as resistance, but it also strikes me that it's also just a good way to find _blind spots_, that is, the things that capitalism is good at seeing are where we spend most of our energy. But what about the things that we want that are in its blind spots? It's like the metaphor of "looking for your keys under all the lamp-posts" because that's where the light is, but at some point, if you've done that for a while, you should start looking exactly in those places wehre there are no lamp-posts.

I think we're all feeling that not enough energy is being spent searching between the lamp-posts, and so it's just a good policy to be this in-appropriateable shape.